cookbooks/bcpc-hadoop/recipes/hbase_master.rb

```diff
@@ -13,6 +13,66 @@
     'docopy' => true
 }
 
+# Define zabbix triggers for alarming
+# The Hbase triggers depend on the namenode triggers too. So the namenode triggers
+# are also defined here. This takes care of creation of triggers irrespective of
+# how namenode and hbase_master appear in the runlist. Assumption is that hdfs will
+# be installed if hbase is installed
+node.default['bcpc']['hadoop']['graphite']['queries']['namenode'] = {
```

Comment:
The problem was, I had included namenode related queries in `namenode_master` recipe which was not in the run list on the VM cluster (default mode is Non-HA). The non-HA mode runs `namenode_no_HA` recipe.